backend/src/Manager/Notification/NotificationLocalManager.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Manager\Notification;
+
+use App\AutoMapping;
+use App\Entity\NotificationLocalEntity;
+use App\Repository\NotificationLocalEntityRepository;
+use App\Request\Notification\NotificationLocalCreateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+use DateTime;
+
+class NotificationLocalManager
+{
+    public function __construct(private AutoMapping $autoMapping, private EntityManagerInterface $entityManager, private NotificationLocalEntityRepository $notificationLocalRepository)
+    {
+    }
+    
+    /**
+     * @param NotificationLocalCreateRequest $request
+     * @return NotificationLocalEntity|null
```

Comment:
No need for functions' annotations while we already define the type of either the functions' parameters and the return type.